### ISSUE TITLE
isInitialized flag

### ISSUE THREAD

**Issue by OlegTarashkevich** [AppLovin/AppLovin-MAX-SDK-Android#35]
Hi dear team! @thomasmso 

I need to know when the sdk is initialised or not from any place of the code and in different threads.

Please add a method for manual check: 

```
AppLovinSdk.isSdkInitialized();
or
AppLovinSdk.getInstance(context).isSdkInitialized();
```

Currently I use:

```
boolean isApplovinInitialized = false;

 AppLovinSdk.getInstance(context).initializeSdk(new AppLovinSdk.SdkInitializationListener() {
                @Override
                public void onSdkInitialized(final AppLovinSdkConfiguration config) {
                    // SDK finished initialization
                    isApplovinInitialized = true;
                    RxBus.getInstance().send(new AdsEvent(AdType.APPLOVIN, AdsEvent.Action.INIT));
                }
            });
```

**But it will be useful to ask the sdk directly.**

Why I need it? Because I see next error:
W/AppLovinSdk: [AppLovinSdk] Attempted to load ad before SDK initialization. Please wait until after the SDK has initialized, e.g. AppLovinSdk.initializeSdk(Context, SdkInitializationListener).

It also strange, because Interstitial ad is loading, but banner shows 204 error.

**Comment by thomasmso** [comment]
Hi @OlegTarashkevich - makes sense, we'll add the new API in the next SDK release (eta ~2-3 weeks since we just did a release last week) and I don't believe you are blocked.

Re: banners - I would try testing with test ads first via the SDK's [Mediation Debugger](https://stage-dash.applovin.com/documentation/mediation/android/testing-networks/mediation-debugger). If you test with live ads, there are no guarantees that ads will load.

**Comment by thomasmso** [comment]
@OlegTarashkevich - We just released Android SDK 10.3.1 which contains the desired API. Let me know if you have any questions. Thanks.

**Comment by OlegTarashkevich** [comment]
@thomasmso Thank you!
Just updated the sdk.
